backend/ccm/models.py

```diff
@@ -1,3 +1,3 @@
 from django.db import models
```

Comment:
Delete this file

Reply:
I did not mean to delete the file, I think since there are no changes to the file, this file shouldn't appear in the PR. Sorry for the confusion